server/services/history.py

```
@@ -1,32 +1,133 @@
+        self.max_file_bytes = max_file_bytes
+        self._write_failure_logged = False
         self.last_cleanup = time.time()
+        if self.persist:
+            self._load_from_disk()
+
+    # ── 落地 ────────────────────────────────────────────────────────────
+
+    def _load_from_disk(self) -> None:
+        """啟動時把檔案內容讀回 deque。
+
+        deque 的 maxlen 會自動只保留最後 max_records 筆，所以即使檔案比記憶體
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)